pfcli/deployment.py

```diff
@@ -68,7 +68,7 @@
 deployment_panel = PanelFormatter(
     name="Deployment Overview",
     fields=[
-        "deployment_id",
+        "id",
```

Comment:
Currently, staging server uses "id", and dev server uses "deployment_id".
Since updates in dev clusters will be applied to staging server soon, we should leave this field as-is.